java/lib/build.gradle.kts

```diff
@@ -1,24 +1,20 @@
 plugins {
-    // Apply the java-library plugin for API and implementation separation.
-    `java-library`
+    `maven-publish` // Apply the maven-publish plugin before java-library for publishing to GitHub Packages.
```

Comment:
Will it be used together with workflow like [this one in the archived repo](https://github.com/ARK-Builders/arklib-android/blob/main/.github/workflows/release.yml)?

Reply:
Exactly. It allows the publish action:
```yml
    - name: Publish Release to Github
      uses: gradle/gradle-build-action@v2
      with:
        arguments: publishReleasePublicationToGithubPackagesRepository
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```